spec/linter-clang-spec.js

```diff
@@ -0,0 +1,48 @@
+"use babel";
+
+provider = require("../lib/main").provideLinter();
+
+describe("The Clang Provider for AtomLinter", () => {
+  beforeEach(() => {
+    waitsForPromise(() => {
+      atom.packages.activatePackage("linter-clang");
+    });
+  });
+
+  describe("finds issue with the code", () => {
+    it("in 'missing_import.c'", () => {
+      waitsForPromise(() => {
+        atom.workspace.open("./files/missing_import.c").then((editor) => {
```

Comment:
You need to return here, or drop the `{}` of the arrow function. Same for others with promises